packages/core/data-transfer/lib/providers/local-file-source-provider.ts

```diff
@@ -82,14 +82,15 @@ class LocalFileSourceProvider implements ISourceProvider {
     // TODO: need to read the file & extract the metadata json file
     // => we might also need to read the schema.jsonl files & implements a custom stream-check
     const backupStream = this.#getBackupStream();
-
-    return this.#parseJSONFile<IMetadata>(backupStream, METADATA_FILE_PATH);
+    return {};
+    // return this.#parseJSONFile<IMetadata>(backupStream, METADATA_FILE_PATH);
   }
 
   async getSchemas() {
-    const schemas = await collect(this.streamSchemas() as Readable);
+    // const schemas = await collect(this.streamSchemas() as Readable);
 
-    return keyBy('uid', schemas);
+    // return keyBy('uid', schemas);
+    return null;
```

Comment:
Shouldn't this provide the schema that is in the file?

Reply:
well, yes and no, this needs work I faced some errors while trying the import, so I commented the stuff that caused the issues, we should probably work on those independently when there task comes up, so far I wanted the restore to work without being affected by outside issues

Reply:
well, one the integrity check will fail because from the otherside we don't get the schema, and the other is related to that the export is compressed and encrypted (sometimes)